build.gradle

```diff
@@ -1,24 +1,22 @@
 plugins {
+    id 'java'
     id 'org.springframework.boot' version '2.2.5.RELEASE'
+    id 'io.spring.dependency-management' version '1.0.8.RELEASE'
```

Comment:
you dont need that with boot

Reply:
Hmm i tried it without and run into errors. 

<img width="1103" alt="image" src="https://user-images.githubusercontent.com/56041579/81942331-4a5a0e80-95fa-11ea-8d1a-3188692b1865.png">

Reply:
check our internal services